apps/api-extractor/src/analyzer/SourceFileLocationFormatter.ts

```diff
@@ -0,0 +1,61 @@
+// Copyright (c) Microsoft Corporation. All rights reserved. Licensed under the MIT license.
+// See LICENSE in the project root for license information.
+
+import * as ts from 'typescript';
+import * as path from 'path';
+import { Path, Text } from '@rushstack/node-core-library';
+
+export interface ISourceFileLocationFormatOptions {
+  sourceFileLine?: number;
+  sourceFileColumn?: number;
+  workingPackageFolderPath?: string
+}
+
+export class SourceFileLocationFormatter {
```

Comment:
It'd be cool to genericize this. Maybe add it to the `Terminal` API?

Reply:
That's not a bad idea. API Extractor's usage is coupled to the compiler API, so I think I would approach this work item from some other tool that wants to report file/line information.  

`Terminal` doesn't seem like the right place, because it's about writing to a terminal. `Path` might not be a bad choice, or maybe something more specific like `SourceFilePath`.  It could possibly also include a generalized equivalent of `getLineAndCharacterOfPosition()`.  

But again, I'd want to introduce that API as part of solving some problem that actually needs it, so we know what the requirements are.